python/python/lance/vector.py

```diff
@@ -182,9 +183,13 @@ def train_ivf_centroids_on_accelerator(
         )
         kmeans.fit(ds)
 
-    return kmeans.centroids.cpu().float().numpy(), compute_partitions(
-        dataset, column, kmeans, batch_size=20480
-    )
+    centroids = kmeans.centroids.cpu().float().numpy()
+
+    file_name = f"/tmp/centroids-{uuid.uuid4().hex}"
```

Comment:
Use tempfile to create the tempfile, some production system the `TMPDIR` is not here.